app.R

```diff
@@ -846,14 +846,14 @@ server <- function(input, output, session) {
   observeEvent(input$getStats, {
     # only want to use things with valid names from POWO
     valid_names <- filter(values$powo_results, is.na(warnings))
-    print("valid_names working")
+    #print("valid_names working")
```

Comment:
I'd remove all lines of code that have been commented out, to tidy things up a bit.